libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Logger/SFSDKLogger.m

```diff
@@ -169,4 +196,21 @@ - (DDLogLevel)readLogLevel {
     return logLevel;
 }
 
+static inline DDLogFlag DDLogFlagForLogLevel(DDLogLevel level) {
+    switch (level) {
+        case DDLogLevelOff:
```

Comment:
You turn it off by setting the `logLevel` property. This is simply returning the associated flag level. It doesn't affect the actual log level.